### ISSUE TITLE
[F#] Highlight usages does not work in VSMac 8.1 preview

### ISSUE THREAD

**Issue by nosami** [mono/monodevelop#7770]
```
DEBUG [2019-06-03 11:30:29Z]: [UI - false] HighlightUsagesExtension: ResolveAsync starting on FSharpTokens.fs
DEBUG [2019-06-03 11:30:29Z]: [UI - false] LanguageService: GetUsesOfSymbolAtLocationInFile: file:FSharpTokens.fs, line:26, col:22
DEBUG [2019-06-03 11:30:29Z]: [UI - false] LanguageService: GetProjectCheckerOptions: Getting ProjectOptions from cache for:MonoDevelop.FSharp.fsproj}
DEBUG [2019-06-03 11:30:29Z]: [UI - false] LanguageService: GetTypedParseResultWithTimeout: No stale results - typechecking without timeout
DEBUG [2019-06-03 11:30:29Z]: LanguageService agent: Exception
System.Exception: The file 'FSharpTokens.fs' was not part of the project. Did you call InvalidateConfiguration when the list of files in the project changed?
  at Microsoft.FSharp.Compiler.IncrementalBuilder.GetSlotOfFileName (System.String filename) [0x00036] in <5b8fc08ead8b7fdaa74503838ec08f5b>:0
  at Microsoft.FSharp.Compiler.IncrementalBuilder.GetCheckResultsBeforeFileInProject (Microsoft.FSharp.Compiler.AbstractIL.Internal.Library+CompilationThreadToken ctok, System.String filename) [0x00000] in <5b8fc08ead8b7fdaa74503838ec08f5b>:0
  at <StartupCode$FSharp-Compiler-Service>.$Service+ParseAndCheckFileInProject@2721-6.Invoke (Microsoft.FSharp.Compiler.AbstractIL.Internal.Library+CompilationThreadToken ctok) [0x00000] in <5b8fc08ead8b7fdaa74503838ec08f5b>:0
  at <StartupCode$FSharp-Compiler-Service>.$Reactor+EnqueueAndAwaitOpAsync@174-2[T].Invoke (Microsoft.FSharp.Compiler.AbstractIL.Internal.Library+CompilationThreadToken ctok) [0x00007] in <5b8fc08ead8b7fdaa74503838ec08f5b>:0

```

> VS bug [#904300](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/904300)

**Comment by xamarin-release-manager** [comment]
> [sync] [VS-13] Comment by **xamarinc**



Fixed in version 8.1.0.2703 (release-8.1)



Pull Request #7794 merged by: Lluis Sanchez  
 Author: mono  
 Commit: 7484f0f8cb0ad176227443d50996202f6d62fa9a (mono/monodevelop)

**Comment by xamarin-release-manager** [comment]
> [sync] [VS-15] Comment by **xamarinc**



Fixed in version 8.3.0.235 (master)



Pull Request #7793 merged by: Lluis Sanchez  
 Author: mono  
 Commit: 2b376dda157137200fdec7ee6630a02b04918335 (mono/monodevelop)

**Comment by xamarin-release-manager** [comment]
> [sync] [VS-16] Comment by **xamarinc**



Fixed in version 8.2.0.653 (release-8.2)



Pull Request #7793 merged by: Lluis Sanchez  
 Author: mono  
 Commit: 2b376dda157137200fdec7ee6630a02b04918335 (mono/monodevelop)

**Comment by xamarin-release-manager** [comment]
> [sync] [VS-20] Comment by **xamarinc**



Fixed in version 8.6.0.802 (master)



Pull Request #7793 merged by: Lluis Sanchez  
 Author: mono  
 Commit: 2b376dda157137200fdec7ee6630a02b04918335 (xamarin/monodevelop)
